package.json

```diff
@@ -20,10 +20,17 @@
   "homepage": "https://github.com/LukarioMC/COMP-4800-37#readme",
   "dependencies": {
     "@prisma/client": "^5.13.0",
+    "@quixo3/prisma-session-store": "^3.1.13",
+    "body-parser": "^1.20.2",
```

Comment:
Don't forget to take out body-parser from here as well please 😄

Reply:
Removed body-parser.